consensus/validator.go

```diff
@@ -276,14 +277,13 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 	consensus.commitBitmap = mask
 
 	if recvMsg.BlockNum-consensus.blockNum > consensusBlockNumBuffer {
-		consensus.getLogger().Debug().Uint64("MsgBlockNum", recvMsg.BlockNum).Msg("[OnCommitted] OUT OF SYNC")
+		utils.Logger().Debug().
+			Uint64("MsgBlockNum", recvMsg.BlockNum).
+			Msg("[OnCommitted] OUT OF SYNC")
 		go func() {
 			select {
 			case consensus.BlockNumLowChan <- struct{}{}:
 				consensus.current.SetMode(Syncing)
-				for _, v := range consensus.consensusTimeout {
```

Comment:
is it safe to remove this part?

Reply:
I think so - its based on a faulty assumption - I can put it back in, but I don't think its a make or break thing - in any case, I need a deeper redo of consensus interacting with state sync, which will get to in next PR  (part of my work in other branch from which this work taken from)